mobile/.bazelrc

```diff
@@ -256,6 +255,9 @@ build:mobile-remote-ci-android --config=mobile-remote-ci
 test:mobile-remote-ci-android --build_tests_only
 test:mobile-remote-ci-android --config=mobile-test-android
 test:mobile-remote-ci-android --config=mobile-remote-ci
+# enabled for //test/kotlin/integration:xds_test and
+# //test/java/io/envoyproxy/envoymobile/engine:envoy_configuration_test
+test:mobile-remote-ci-android --define=envoy_mobile_xds=enabled
```

Comment:
should we set it for `mobile-test-android` instead?  that way, we will get xDS enabled if we run the tests locally too.